uint8 ID
uint32 time
float32 target_pressure
float32 feedback_pressure
float32 error
float32 smc_uc
int32 valve_mode
